src/wg/color/package.d

```diff
@@ -1,22 +1,28 @@
-module wg.color;
+// Written in the D programming language.
+/**
+RGB color space.
 
+Authors:    Manu Evans
+Copyright:  Copyright (c) 2019, Manu Evans.
+License:    $(WEB boost.org/LICENSE_1_0.txt, Boost License 1.0)
```

Comment:
We could as well add the license file with this PR.

Reply:
Let's do that separately. We will need to talk about e.g. image test suites licensing which won't be Boost.

Reply:
Fair enough.

Reply:
Please add copyrights, but honestly Authors is a bit redundant with copyright and many project don't provide it.

Reply:
I based it upon what is in Phobos in terms of what sections should be provided. Most of what I can see, Authors section is used for links, where as Copyright is much more formal legal wise.

My preference is using Phobos as the standard for documentation. But either way, it can always be removed and we need to start clearing the backlog or PRs.

Although I can understand the view point of it being a bit redundant in this case. Hopefully it won't be in the future, once more people are contributing.